### DISCUSSION TITLE
Pore Water Pressure

### DISCUSSION THREAD

**Discussion by nubufi** [Caltech-geoquake/PySeismoSoil#15] [General]
It would be perfect if you can add pore water pressure effect to your calculations.

**Comment by jsh9** [comment]
Hi @nubufi , sorry for the very delayed reply!

It may not be in the roadmap of PySeismoSoil to implement this, because the underlying constitute model does not take into account pore water pressure.

**Reply by dhper** [reply]
It depends on which models we are talking about. It would likely be easy to implement the GMP model developed by Green et al. (2000). This simple model 'relates pore pressure generation to the energy dissipated within the soil' and can be used to evaluate the liquefaction potential of sands.

Green RA, Mitchell JK and Polito CP 2000. "An Energy-Based Pore Pressure Generation Model for Cohesionless Soils", Proceedings: John Booker Memorial Symposium, Melbourne, Australia, November 16-17, 2000. 

Polito C, Green RA, Dillon E and Sohn, C 2013. Effect of load shape on relationship between dissipated energy and residual excess pore pressure generation in cyclic triaxial tests. Canadian Geotechnical Journal, 50: 1118–1128.
